src/store/modules/shipment/actions.ts

```diff
@@ -0,0 +1,80 @@
+import { ShipmentService } from "@/services/ShipmentService";
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import ShipmentState from './ShipmentState'
+import * as types from './mutation-types'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+import emitter from '@/event-bus'
+
+
+const actions: ActionTree<ShipmentState, RootState> = {
+  async findShipment ({ commit, state }, payload) {
+    let resp;
+    try {
+      resp = await ShipmentService.fetchShipments({
+        "viewSize": payload.viewSize,
+        "viewIndex": payload.viewIndex,
+        "facilityId": payload.facilityId,
+        "statusId": "PURCH_SHIP_SHIPPED"
+      })
+      if (resp.status === 200 && resp.data.count> 0 && !hasError(resp)) {
+        let shipments = resp.data.docs;
+
+        if (payload.viewIndex && payload.viewIndex > 0) shipments = state.shipments.list.concat(shipments)
+        commit(types.SHIPMENT_SEARCH_UPDATED, { shipments: shipments })
```

Comment:
Just pass `shipments`
```suggestion
        commit(types.SHIPMENT_SEARCH_UPDATED, { shipments })
```

Reply:
Done